tests/tests.ts

```diff
@@ -88,6 +92,10 @@ describe('Transpose a Dockerfile', () => {
 			stripIndents`
 			FROM ubuntu
 			COPY ["${opts.hostQemuPath}","${opts.containerQemuPath}"]
+			ARG foo=bar zip
+			ARG myVar
+			EXPOSE 80 123/udp
+			EXPOSE 8080
```

Comment:
Before the fix, the problem was that, for example, an input line of `EXPOSE 8080` would produce an incorrect output of `EXPOSE [8080]`.